src/plugins/saved_search/server/saved_objects/search.ts

```diff
@@ -62,6 +63,13 @@ export function getSavedSearchObjectType(
           create: SCHEMA_SEARCH_MODEL_VERSION_3,
         },
       },
+      4: {
+        changes: [],
```

Comment:
I don't think a `change` is necessary here since it's only a validation schema change. These are the current supported change types and they're all based around mapping changes or data modification which isn't needed here: https://github.com/elastic/kibana/blob/e2ccaf2c5305b7c3843d77a7aed20306eb0bf73f/packages/core/saved-objects/core-saved-objects-server/src/model_version/model_change.ts#L30-L35